docs/pages/kubernetes-access/introduction.mdx

```diff
@@ -1,13 +1,12 @@
 ---
 title: Kubernetes Access
-description: Teleport Kubernetes Access introduction, demo and resources
+description: Teleport Kubernetes Access introduction, demo, and resources.
 ---
 
-Teleport provides a unified access for Kubernetes clusters.
+Teleport provides unified access for Kubernetes clusters.
 
-- Users can receive short-lived certificates using Single Sign-On (SSO) and switch between clusters without relogins.
-- Admins can use *roles* to implement policies like the best practice that *developers must not access production* and enforce
-  dual authorization using [access requests](../enterprise/workflow/index.mdx) for privileged operations.
+- Users can receive short-lived certificates using Single Sign-On (SSO) and switch between clusters without logging in twice.
```

Comment:
```suggestion
- Users can receive short-lived kubeconfigs using Single Sign-On (SSO) and switch between clusters without logging in twice.
```

Reply:
Went with `- Users can receive short-lived kubeconfigs and certificates using Single Sign-On (SSO) and switch between clusters without logging in twice` since Kubernetes uses both (although in different circumstances). I think that keeps the intent of your suggestion.